.github/workflows/ci-linux.yml

```diff
@@ -8,29 +8,22 @@ jobs:
     steps:
     - name: Checkout source
       uses: actions/checkout@v3
-    - name: Install system dependencies
+    - name: Install System Packages
       run: |
-        wget -qO - https://packages.lunarg.com/lunarg-signing-key-pub.asc | sudo tee /etc/apt/trusted.gpg.d/lunarg.asc
```

Comment:
Do you want to skip Vulkan SDK installation for now?

Reply:
Yeah because we did not use it yet and it seems like we do not need it soon.

Reply:
That's fine. These steps will stay in Git history anyway

Reply:
Right. When we need it we can grab this from the history without required to keep maintenance it while we are not using.